rero_ils/theme/templates/security/email/reset_instructions.html

```diff
@@ -18,4 +18,14 @@
 
 #}
 
-<p><a href="{{ reset_link.replace('/api', '') }}">{{ _('Click here to reset your password') }}</a></p>
\ No newline at end of file
+{%- if user.profile.first_name and user.profile.last_name %}
+<p>{{ _('Dear') }} {{ user.profile.first_name }} {{ user.profile.last_name }},</p>
+{%- else %}
+<p>{{ _('Dear patron') }},</p>
```

Comment:
Is this has been validate by the PO? Sounds wired to me.

Reply:
Yes